<!---->
<launch>

<!--read goals from person follower and publish them for the laser_scanner_node-->
	<node pkg="goal_manager" type="robot_goal_manager" name="robot_goal_mangager" output="screen">
		<param name="next_point" value="/point"/>
		<param name="next_pose" value="/pose"/>
		<param name="waypoint_list" value="$(find fieldrobot_05)/waypoints/demo.txt"/>
		<param name="path_publisher" value="/robot_path"/>
		<!--as lower frequency as longer the robot will wait at a reached goalpoint-->
		<param name="frequency" value="10"/>
		<param name="user_goal_input" value="/move_base_simple/goal"/>
		<param name="rabbit_frame" value="rabbit"/>
		<param name="robot_frame" value="base_link"/>
		<param name="fixed_frame" value="odom"/>
		<param name="waittime" value="0.1"/>
		<param name="min_distance" value="0.1"/>
		<param name="distance" value="0.1"/>		
		<param name="wait_for_topic" value="true"/>
		<param name="wait_topic_name" value="/joy/Button_B"/>
		<param name="user_goal_input" value="/move_base_simple/goal"/>
		
	</node>

<!--check the laser pointcloud for emergency stops and give back how to turn at the place--> 
	<node pkg="fieldrobot_05" type="laser_error_cmd" name="laser_error_cmd" output="screen">
		<param name="front_cloud" value="/error_cloud_front"/>
		<param name="back_cloud" value="/error_cloud_back"/>
		<param name="front_left" value="/front_left"/>
		<param name="front_right" value="/front_right"/>		
		<param name="back_left" value="/back_left"/>
		<param name="back_right" value="/back_right"/>
		<param name="cmd_vel_pub" value="/cmd_vel_obstacle"/>
		<param name="mode_pub" value="/mode_obstacle"/>
		<param name="mode_changer_active" value="true"/>
		<param name="point_ignore" value="3"/>
	</node>

	
</launch>
